docs/02-google-sheet.md

```diff
@@ -13,15 +13,33 @@ https://docs.google.com/document/d/<ID>/edit
 
 Note that there are different sheets for different environments.
 
-## Interpreting the Google sheet.
+## The sheet format
+
+An example rule sheet with a few instances of rules of different types is available [here](https://docs.google.com/spreadsheets/d/1n5xjfVnvRQBMfmjD_VzFX2ye4pLBmL98whdDqsJhtAs).
```

Comment:
Checked and confirmed I can open that link in an incognito window.

Reply:
If you've only got read access, then that's working as intended – this is a public repo, and this PR has been added to improve onboarding for non-gu users.